src/ert/analysis/_es_update.py

```diff
@@ -214,7 +213,7 @@ def analysis_IES(
     updatestep: "UpdateConfiguration",
     obs: "EnkfObs",
     rng: np.random.Generator,
-    module_config: IesConfig,
+    module: "AnalysisModule",
```

Comment:
I still don't know much about python type hints, and I again sense much inconsistency in our huge code base, but I wonder if the quotation marks around the type are necessary here.

Reply:
Probably as necessary as those for EnkfObs :laughing:  I will try it without them

Reply:
hmmm trying to read up on it, but i didn't get far, it's seems to be something about forward dependency. which i don't really understand : (

so i'll just quietly resolve this : )